sign_test.go

```diff
@@ -19,10 +18,8 @@ import (
 func TestSign(t *testing.T) {
 	content := []byte("Hello World")
 	sigalgs := []x509.SignatureAlgorithm{
-		x509.SHA1WithRSA,
```

Comment:
The two removed algorithms in this list cause tests to fail with `cannot verify signature: insecure algorithm SHA1-RSA (temporarily override with GODEBUG=x509sha1=1)" while trying to verify candidate authority certificate`. We can add `GODEBUG=x509sha1=1` to the `test` recipe or remove these. Curious to hear what you think.

Reply:
I'd propose adding something like https://github.com/sigstore/cosign/blob/77206c440896783170f144f21078215eaf17ec37/.github/workflows/tests.yaml#L80-L82 to the CI GHA. I think we'll need to continue to test sha1 because this still targets go 1.13